music.js

```diff
@@ -1,16 +1,21 @@
 // Define the Artist constructor function here, with a 'name' property that can be set at instantiation,
 // and a 'songs' property that is an empty array to start
-function Artist() {
-
+function Artist(name, songs) {
+  this.name = name;
+  this.songs = [];
 }
 
 // Define the Song constructor function here, with 'title' and 'artist' properties that can be set at instantiation,
 // and a 'playCount' property that is an empty array to start
-function Song() {
+function Song(title, artist) {
+  this.title = title;
+  this.artist = artist;
+  this.playCount = []
+}
 
   // When a new song is instantiated, you'll want to add that song to its artist's 'songs' array.
   // You'll probably want to do that in here somewhere...
-}
+songs.push(Song);
```

Comment:
Should be in the `Song` function constructor, and it should be as follows:

`this.artist.songs.push(this)`

Let me know if you have any questions